src/lib/i18n.tsx

```diff
@@ -104,19 +36,5 @@ export function Provider({ locale, forceRenderAfterLocaleChange = true, onActiva
       })
   }, [locale, onActivate])
 
-  // Initialize the locale immediately if it is DEFAULT_LOCALE, so that keys are shown while the translation messages load.
-  // This renders the translation _keys_, not the translation _messages_, which is only acceptable while loading the DEFAULT_LOCALE,
-  // as [there are no "default" messages](https://github.com/lingui/js-lingui/issues/388#issuecomment-497779030).
-  // See https://github.com/lingui/js-lingui/issues/1194#issuecomment-1068488619.
-  if (i18n.locale === undefined && locale === DEFAULT_LOCALE) {
-    i18n.loadLocaleData(DEFAULT_LOCALE, { plurals: () => plurals[DEFAULT_LOCALE] })
-    i18n.load(DEFAULT_LOCALE, {})
-    i18n.activate(DEFAULT_LOCALE)
-  }
-
-  return (
-    <I18nProvider forceRenderOnLocaleChange={forceRenderAfterLocaleChange} i18n={i18n}>
-      {children}
-    </I18nProvider>
-  )
+  return <I18nProvider i18n={i18n}>{children}</I18nProvider>
```

Comment:
Test that this will in fact change macro'ed translations when a user changes their locale.

Reply:
This does not work, and is in fact a large regression in their major version bump, but it doesn't matter because we only show one translated macro (the search bar) in the page. Any others are hidden by the language selector.

Reply:
Are you able to load translations at all? I compiled our i18n however switching languages via the settings menu is not changing any text for me, not just the search placeholder

Reply:
I downloaded one locale explicitly from crowdin to test this. It's not testable via CI until it hits the staging branch.

Reply:
There is a workaround I've been using locally to test localization which is to merge `releases/staging` into my local branch to get the translations and then I'm able to switch languages. I've tried this workaround with this PR however I'm still unable to see translations. Could you add more details about how you tested this by downloading locales?

Reply:
go to our console's translations: https://crowdin.com/project/uniswap-interface/translations
download one language (I chose fr-FR) through the "Build & Download" option
mv the file into src/locales
run `yarn i18n:compile` to generate the necessary js messages
`yarn start` or `yarn build && yarn serve`
test

Reply:
okay successfully verified that compiled translations loaded 👍